kit/wifi.md

```diff
@@ -6,69 +6,59 @@ title: WiFi
 WiFi
 ====
 
-The WiFi dongles provided with your kit allow you to interface with and debug your robot.
+The Raspberry Pi that makes up your kit's brain board contains a WiFi radio which allows you to interface with and debug your robot.
 You can connect to your robot using any WiFi capable device (laptop, tablet, phone, etc.)
 
 Connecting to Your Robot
 ------------------------
 
-First, ensure that the WiFi dongles supplied with your kit are plugged into your robot.
-Once you have the WiFi dongles connected, plug in a USB memory stick and turn your robot on.
-
-During the boot process you should notice a green LED begin to flash on one of the WiFi dongles.
-Your robot has now set up its very own WiFi network! It'll be called `robot-XXX`, where `XXX` is
-your team's TLA.
+During the boot process you should notice the green LED labelled WiFi turn on.
+Your robot has now set up its very own WiFi network! It will have a name starting with `robot-ZZZ` followed by some random numbers.
```

Comment:
Question: What determines "ZZZ"? Is it their TLA? Also, are the random numbers deterministic between reboots?

Reply:
The number is deterministic, it gets saved into `robot-settings.toml` on first run.

Reply:
480171c